service-registry/pom.xml

```diff
@@ -71,5 +71,19 @@
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-lang3</artifactId>
     </dependency>
+    <dependency>
+        <groupId>io.swagger</groupId>
+        <artifactId>swagger-models</artifactId>
+        <version>1.5.22</version>
```

Comment:
do not add version, use dependency management if neccessary

Reply:
done